### PR TITLE
Move all Edge and CloudN resources and data sources under Edge category.

### PR DESCRIPTION
**Pull request by Dennizz**
This reorganizes all Edge and CloudN resources and data sources to be under the Edge category. This improves navigation through our resources on the provider documentation page.

### CODE REVIEW

**Comment by inc1t3Ful** [comment]
Good change! I think we should merge this in after our upcoming release this week. We will likely need a backport for this too once merged into master

**Review by dnardo — APPROVED** [review]

**Pull request merged**
